server/config/toml.go

```diff
@@ -176,6 +176,14 @@ enable = {{ .GRPC.Enable }}
 # Address defines the gRPC server address to bind to.
 address = "{{ .GRPC.Address }}"
 
+# MaxRecvMsgSize defines the max message size in bytes the server can receive.
+# The default value is 4MB.
+max-recv-msg-size = "{{ .GRPC.MaxRecvMsgSize }}"
+
+# MaxSendMsgSize defines the max message size in bytes the server can send.
+# The default value is math.MaxInt32.
```

Comment:
asking to learn - what is the reason behind the choice of this default for the send and only 4MB for the receive?

Reply:
It's actually the gRPC default. In fact, the entire comment is more or less copied from there.

Ref: https://github.com/grpc/grpc-go/blob/a73725f42db97964eb538a5cf4d302f760357dbe/server.go#L58-L59